cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOriginKey.ds

```diff
@@ -79,7 +80,16 @@ function getOriginKey(args) {
     return responseObj;
 }
 
+function getOriginKeyFromRequest(protocol, host) {
+	var origin = protocol + "://" + host;
+	var originKeys = getOriginKey(origin);
```

Comment:
This is bit confusing. shall we call the variable as OriginKeysResponse and then getting originekeys from response:
return originKeysResponse["originKeys"][origin];